tests/graphql_spec.rs

```diff
@@ -23,16 +23,54 @@ mod graphql_mock;
 
 static INIT: Once = Once::new();
 
+#[derive(Debug, Clone)]
+struct Source {
+  sdl: String,
+  tag: String,
+}
+
 #[derive(Debug, Default, Setters)]
 struct GraphQLSpec {
   path: PathBuf,
-  client_sdl: String,
-  server_sdl: Vec<String>,
-  merged_server_sdl: String,
+  sources: Vec<Source>,
   sdl_errors: Vec<SDLError>,
   test_queries: Vec<GraphQLQuerySpec>,
 }
 
+impl GraphQLSpec {
+  fn client_sdl(mut self, sdl: String) -> Self {
```

Comment:
can also use `get_sources` instead.

Reply:
sure, I'll use `get_sources`; the tag is not `"client-sdl"`?

Reply:
this is a getter and not a setter.

Reply:
I'm using `get_sources` on the getter as suggested; I'm resolving this.

if you have any further comments, let me know

Reply:
Why do we need a setter?

Reply:
we used a setter before but via directives; I'm keeping the same approach as before (I chose this way to avoid significant changes), but it is not required

Reply:
@ologbonowiwi I think we can remove the setters.

Reply:
done @tusharmath